packages/viewer/src/components/scene-tree/scene-tree.tsx

```diff
@@ -184,6 +184,13 @@ export class SceneTree {
   @Prop({ mutable: true })
   public controller?: SceneTreeController;
 
+  /**
+   * Temporary flag to indicate whether metadata should be used when filtering
+   * as opposed to just the name of the item.
+   */
+  @Prop({ mutable: true })
```

Comment:
`mutable` only needs to be set if the property is modified within the component.

Reply:
Noted, thanks!